src/locale_EN.h

```diff
@@ -87,13 +87,14 @@ MAKE_PSTR_WORD(boiler)
 MAKE_PSTR_WORD(thermostat)
 MAKE_PSTR_WORD(switch)
 MAKE_PSTR_WORD(solar)
-MAKE_PSTR_WORD(mixing)
+MAKE_PSTR_WORD(mixer)
 MAKE_PSTR_WORD(gateway)
 MAKE_PSTR_WORD(controller)
 MAKE_PSTR_WORD(connect)
 MAKE_PSTR_WORD(heatpump)
 MAKE_PSTR_WORD(generic)
 MAKE_PSTR_WORD(dallassensor)
+MAKE_PSTR_WORD(sensor)
```

Comment:
if we just use dallassensor everywhere we can drop this

Reply:
If you also use it for sensor-numbering it could be to long for mobile view. Have to test.

![Screenshot_20201025-123400](https://user-images.githubusercontent.com/59284019/97105977-2c232f00-16bf-11eb-9085-33b0d1e8273c.png)

Reply:
> Also should we consider renaming system to system_data in MQTT ?

´system` is a command subscription like `boiler` and `thermostat` and the name fits. 
We dont have system_data published, they are all in `heartbeat`. 
You can consider to publish in heartbeat only a small changing value (uptime_sec) and publish the rest in system_data.

Reply:
@MichaelDvP looking here at the web UI it says "Sensor #" so ideally we should just be showing the number in the UI (1, 2,...) and skip the word "Sensor" ?